state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -1044,13 +1048,14 @@ pub mod pallet {
 				from,
 				to,
 				amount,
+				net_amount,
 				destination_address.clone(),
 				&swap_origin,
 			) {
 				Self::deposit_event(Event::<T>::SwapScheduled {
 					swap_id,
 					source_asset: from,
-					deposit_amount: amount,
+					deposit_amount: net_amount,
```

Comment:
```suggestion
					deposit_amount: amount,
```

The event's deposit_amount should match the amount deposited.

Reply:
I don't think this is right. We want to emit the same amount we did before, right? Or was this also wrong before? `amount` is now the raw amount and `net_amount` is `amount - fees`.

Reply:
I think it makes more sense to emit the amount deposited in `deposit_amount` (ie. it was wrong before). But maybe this also depends on how it's used in teh swapping app. @acdibble ?